select.py

```diff
@@ -29,3 +37,20 @@ def current_option(self) -> str:
     async def async_select_option(self, option: str) -> None:
         await self.coordinator.product.maybe_set_mode_from_display(option)
         await self.coordinator.async_request_refresh()
+
+
+# Nuova classe per il sensore della temperatura esterna (AIR_OUTSIDE_TPT)
+class AldesProductOutsideTempSensor(AldesProductEntity, SensorEntity):
+
+    _attr_icon = "mdi:thermometer"
+
+    def __init__(self, coordinator: AldesProductDataUpdateCoordinator) -> None:
+        super().__init__(coordinator, 'Outside Air Temperature')
+
+    @property
+    def native_value(self):
+        return self.coordinator.product.air_outside_tpt  # Restituisce il valore della temperatura esterna
```

Comment:
`air_outside_tpt` is not present in `AldesProduct`, you must provide a way to get this value.